TundraTD/Assets/Scripts/Level/MobPool.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Mobs;
+using Mobs.MobsBehaviour;
+using Mobs.MobsBehaviour.Bear;
+using Mobs.MobsBehaviour.Boar;
+using Mobs.MobsBehaviour.Spider;
+using UnityEngine;
+
+namespace Level
+{
+    public class MobPool : MonoBehaviour
+    {
+        private List<MobBehaviour> _mobsOnLevel;
+
+        private static int _boarStartIndexMarker;
+        private static int _bearStartIndexMarker;
+        private static int _spiderStartIndexMarker;
```

Comment:
Can you please explain these static fields? I guess when there will be more mob types, this system won't work whatever it did